sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -527,6 +530,54 @@ public async Task CreateIdentityWithDifferentServiceVersions(ServiceVersion vers
             }
         }
 
+        [Test]
+        public async Task CreateUserWithCustomIdShouldReturnExistingIdentity()
+        {
+            try
+            {
+                var customId = "bob@contoso.com";
+                CommunicationIdentityClient client = CreateClient();
+                Response<CommunicationUserIdentifier> createResponse = await client.CreateUserAsync(customId);
+
+                Assert.IsTrue((int)HttpStatusCode.Created == createResponse.GetRawResponse().Status);
+                Assert.IsNotNull(createResponse.Value.Id);
+
+                Response<CommunicationUserIdentifier> getResponse = await client.CreateUserAsync(customId);
+                Assert.AreEqual((int)HttpStatusCode.Created, getResponse.GetRawResponse().Status);
+                Assert.AreEqual(createResponse.Value.Id, getResponse.Value.Id);
+            }
+            catch (Exception ex)
```

Comment:
Wouldn't the test framework fail the test anyways if the method throws? Not sure what the value in catching is